.buildkite/beta-builds.yml

```diff
@@ -15,11 +15,11 @@ steps:
     artifact_paths:
       - "**/build/reports/lint-results*.*"
 
-  # - label: "🔬 Lint Jetpack"
-  #   command: ".buildkite/commands/lint.sh jetpack"
-  #   key: jplint
-  #   artifact_paths:
-  #     - "**/build/reports/lint-results*.*"
+  - label: "🔬 Lint Jetpack"
```

Comment:
Minor (🔍): Consider adding this `🔬 ` prefix to the `Lint WordPress` label above, or else, remove that from both. However, since I am also seeing this `🛠 ` prefix added to the `WordPress/Jetpack Beta Build` labels further below, I am thinking we would like to have it... not sure, it is up to you! 🤔 

PS: Same comment applies to the `release-build.yml` file changes.

Reply:
Way nicer indeed! [472c6d6b36]
<img width="295" alt="image" src="https://user-images.githubusercontent.com/216089/183438595-6a369921-e80c-41d8-a1b4-f0d96e9671df.png">

Reply:
Looooove it! ❤️ 🌟 🙇